blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -109,6 +104,8 @@ class RunDetailsHeader extends Component {
     }
 }
 
+const { object, func } = PropTypes;
```

Comment:
We do not use ```Flow``` in dashboard the other is your personal style, I much prefer it like that makes is much better to follow the code.